django/chat/templates/chat/components/chat_security_label.html

```diff
@@ -12,14 +10,12 @@
   <ul class="dropdown-menu">
     <li class="dropdown-header">{% trans "Set security label" %}</li>
     {% for security_label in security_labels %}
- 
       <li>
         <button class="dropdown-item"
                 hx-get="{% url 'chat:set_security_label' chat.id security_label.id %}"
-                hx-target="closest div.dropdown"
+                hx-target="closest .dropdown"
                 hx-swap="outerHTML"
-                onclick="this.closest('div.dropdown').querySelector('button').innerHTML = '...';"
```

Comment:
I did this as an indicator your click did something, in case the server takes a second to respond, but it seems to consistently respond quickly now. All good.